packages/browserid-verifier/README.md

```diff
@@ -1,5 +1,3 @@
-[![Docker Build Status](https://circleci.com/gh/mozilla/browserid-verifier/tree/master.svg?style=shield&circle-token=da8b9c7c4c778b238eeadb6d8bc695f2be4e22df)](https://circleci.com/gh/mozilla/browserid-verifier/tree/master)
```

Comment:
Is this removal on purpose?

Reply:
Good question. That link pointed to the old pre-monorepo location of the browserid-verifier, and it seemed like the "build passing" badge might not reflect current reality. Should I keep that line, though?